### PR TITLE
Fix startup script call to elasticsearch-cli

### PR DESCRIPTION
**Pull request by jkakavas**
In `elasticsearch` script, we source `elasticsearch-env` that ends
with a `cd` command to $ES_HOME. As such we don't need to call
`dirname` to determine where `elasticsearch-cli`, but reference it
with the relevant path `bin/elastisearch-cli`

### CODE REVIEW

**Review by williamrandolph — APPROVED** [review]
You may want to wait for Ryan's approval, but this looks good to me. This script assumes that it can call scripts in `bin/` on line 42 as well, so this change will add consistency.

**Review by rjernst — APPROVED** [review]
LGTM

**Pull request merged**
